numba/tests/test_listobject.py

```diff
@@ -36,6 +36,10 @@ def foo(n):
         for i in (0, 1, 2, 100):
             self.assertEqual(foo(i), i)
 
+    def test_list_create_no_jit(self):
```

Comment:
Whilst this checks the code path its not actually demonstrating the use of `NUMBA_DISABLE_JIT=1` ?